src/components/Sidebar/ConnectNodeButton.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState, memo, useCallback, ChangeEvent, FC } from 'react';
+import { SidebarButton } from 'components/Sidebar';
+import DropDown from 'components/elements/DropDown';
+import useStyles from './styles';
+import { addTransition } from 'utils/builder';
+import { usePathwaysContext } from 'components/PathwaysProvider';
+import { getConnectableNodes } from 'utils/nodeUtils';
+import { useCurrentPathwayContext } from 'components/CurrentPathwayProvider';
+import { useCurrentNodeContext } from 'components/CurrentNodeProvider';
+
+import { faLevelDownAlt } from '@fortawesome/free-solid-svg-icons';
+import { Button } from '@material-ui/core';
+import Tooltip from '@material-ui/core/Tooltip';
+
+const ConnectNodeButton: FC = () => {
+  const { pathway, pathwayRef } = useCurrentPathwayContext();
+  const { currentNode, currentNodeRef } = useCurrentNodeContext();
+  const { updatePathway } = usePathwaysContext();
+
+  const styles = useStyles();
+  const [open, setOpen] = useState(false);
+
+  let options: Array<{ label: string; value: string }> = [];
+  if (currentNode && pathway) options = getConnectableNodes(pathway, currentNode);
+  const optionsAvailable = options.length > 0;
+
+  const connectToNode = useCallback(
+    (event: ChangeEvent<{ value: string }>): void => {
+      const nodeKey = event?.target.value;
+      if (pathwayRef.current && currentNodeRef.current)
+        updatePathway(addTransition(pathwayRef.current, currentNodeRef.current.key ?? '', nodeKey));
```

Comment:
Adding a transition from '' to nodeKey doesn't make sense so add it to the if statement
```suggestion
      if (pathwayRef.current && currentNodeRef.current?.key)
        updatePathway(addTransition(pathwayRef.current, currentNodeRef.current.key, nodeKey));
```